tests/__tests__/csv-generation.test.js

```diff
@@ -0,0 +1,5 @@
+const generate = require('../../database/data-generation.js');
+
+test('the generateData function should return when it has written the appropriate number of items to the specified file', () => {
+	expect(generate.generateData(10, 0, 'test.json')).toBe('Finished Writing Data');
+});
```

Comment:
IMO I think returning a string just for testing purpose should be avoided. Perhaps you could read and check the generated file instead